src/Compilers/CSharp/Test/Symbol/Symbols/Source/FileModifierTests.cs

```diff
@@ -3661,6 +3670,11 @@ class C { }
         var sourceType = comp.GetTypeByMetadataName("<file1>F96B1D9CB33A43D51528FE81EDAFE5AE31358FE749929AC76B76C64B60DEF129D__C");
         Assert.Equal(sourceMember, sourceType);
 
+        var sourceTypeCByMetadataName = comp.GetTypeByMetadataName("C");
+        Assert.NotNull(sourceTypeCByMetadataName);
+        Assert.Equal("C", sourceTypeCByMetadataName!.MetadataName);
```

Comment:
>sourceTypeCByMetadataName! [](http://example.com/codeflow?start=26&length=26)

It is better to suppress nullability analysis in the file. It doesn't bring much value in tests.